fixtures/sync-res.xml

```diff
@@ -1,8 +1 @@
-<?xml version='1.0'?>
-<methodResponse>
-  <params>
-    <param>
-      <value><boolean>1</boolean></value>
-    </param>
-  </params>
-</methodResponse>
```

Comment:
Same for this one. I don't understand why we're moving from a readable approach to oneliners.